src/data/community-meetups.json

```diff
@@ -328,5 +328,11 @@
     "emoji": ":us:",
     "location": "San Diego",
     "link": "http://cryptosandiego.org"
+  },
+  {
+    "title": "Ethereum and Web3 Copenhagen",
+    "emoji": "denmark",
```

Comment:
Need to wrap in colons for the shortcode to work

<img width="1061" alt="Screenshot 2022-11-28 at 13 29 57" src="https://user-images.githubusercontent.com/62268199/204289854-b78b3dae-4464-430e-9d94-6f3a7325de88.png">


```suggestion
    "emoji": ":denmark:",
```